Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Views/CustomPropertyView.xaml

```diff
@@ -6,7 +6,7 @@
         <StackLayout Orientation="Vertical">
             <StackLayout Orientation="Horizontal">
                 <Label Text="Key" HorizontalOptions="Start" VerticalOptions="Center" WidthRequest="60" />
-                <Entry HorizontalOptions="FillAndExpand" x:Name="PropertyKey" />
+                <Entry HorizontalOptions="FillAndExpand" x:Name="PropertyKey" TabIndex="2"/>
```

Comment:
What about Contoso.Forms.Puppet?
Why don't we update nugets now that there is a workaround?